### ISSUE TITLE
areaWidth, areaHeight being ignored...?

### ISSUE THREAD

**Issue by PeterBowater** [PierreCapo/treeviz#33]
Hi!

Been having a play with this, but no matter what I do, I can't get the canvas size to be any different from 800x500 as the default is.

Any ideas? I couldn't see anything in the code immediately.

Expected usage:
```
<TreevizReact
        className='machineView'
        data={data}
        idKey={'id'}
        relationnalField={'father'}
        nodeWidth={130}
        nodeHeight={60}
        areaHeight={300}
        areaWidth={1000}........
```

Thanks,

Peter.

**Comment by PierreCapo** [comment]
Hello Peter, 
I'll try to look at this tonight, it's probably an easy fix. I keep you updated.

Thanks for the report 😀

**Comment by PierreCapo** [comment]
@PeterBowater Hello,
I haven't been able to reproduce your bug on my codebase, both properties are working well.
I have changed the configuration of the whole repository and added an example react app with treeviz-react in it:

To make it run:
1. Clone this repository
2. 
```
cd ./packages/treeviz-react
yarn
cd ./example
yarn
yarn start
```

With that you should see a react app running properly and both props working well.
Please check this example folder or if you are still stuck, link me to a demo codebase which showcase your bug.

FYI: I have just published a new version of `treeviz-react`: 0.4.0 which reflects the change of the new codebase architecture, but the code has remained untouched so I don't think it will fix your issue, but you can have a trying .

**Comment by PeterBowater** [comment]
Hi Pierre,

Thanks so much for looking into this.

Just fired up my dev machine this morning, and preemptively did an npm update.

I'm somewhat scratching my head this morning if, as you say, the code hasn't changed, as it is now working perfectly(!)

I also fired up a starter nextjs app and installed treeviz-react as a bare bones testbed, and that also worked when setting the width and height.

So, thank you very much, for whatever you did, it works now! :)


Peter.
